ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -2539,13 +2545,17 @@ pub struct DirectCallInst {
     args_idx: ArgsIdx,
     /// How many arguments in [Module::args] is this call passing?
     num_args: u16,
+    /// If this is a call to an idempotent function, then the index of the constant containing the
```

Comment:
This comment doesn't quite parse. I think it's trying to say:
> The index of the constant we can replace this function with, iff this function is idempotent and the call's arguments are proven to be constant.

Reply:
I think that the comment is trying to do too much and we should get rid of the second clause. So something like "If this is a call to an idempotent function, then this will contain a [ConstIdx] referencing the return value" might be better?

Reply:
Force pushed a fix